codegen/Cargo.toml

```diff
@@ -9,6 +9,7 @@ homepage = "https://github.com/wasmcloud/weld"
 repository = "https://github.com/wasmcloud/weld"
 documentation = "https://docs.rs/weld-codegen"
 readme = "README.md"
+rust-version = "1.57.0"
```

Comment:
Should this be 1.58.1 to match wasmbus-rpc or is it okay being earlier?

Reply:
It should be what it is. "cargo msrv list" scans all your deps and tells you the max value.